File: JavaScript/grains.js
const square = (squareNum) => {
    if (squareNum < 1 || squareNum > 64) {
        throw new Error("Invalid square");
    }
    if (squareNum === 1) {
        return 1n;
    }
    return 2n ** BigInt(squareNum - 1);
};

const total = () => {
    return 2n ** 64n - 1n;
};

for (let i = 1; i < 65; i++) {
    console.log(`${i} -> ${(0, square)(i)}`);
}

console.log((0, total)());

File: C#/Grains/Grains.cs
using System;
using System.Numerics;

public static class Grains
{
    public static ulong Square(int n)
    {
        return n switch
        {
            < 1 or > 64 => throw new ArgumentOutOfRangeException("Invalid chess board square"),
            1 => 1,
            _ => (ulong)BigInteger.Pow(2, n - 1)
        };
    }

    public static ulong Total()
    {
        return (ulong)(BigInteger.Pow(2, 64) - new BigInteger(1));
    }
}